carma_cloud_client/launch/carma_cloud_client_launch.py

```diff
@@ -33,8 +33,8 @@
 def open_tunnels():
 
     REMOTE_USER="ubuntu"
-    REMOTE_ADDR="carma-cloud.com" # Dev instance is www.carma-cloud.com, prod instance is carma-cloud.com
-    KEY_FILE="carma-cloud-1.pem" # Dev key is carma-cloud-test-1.pem, prod key is carma-cloud-1.pem
+    REMOTE_ADDR="www.carma-cloud.com" # Dev instance is www.carma-cloud.com, prod instance is carma-cloud.com
```

Comment:
For the official release I think we should use the prod instance for both `REMOTE_ADDR` and `KEY_FILE`